src/plugins/controls/public/control_group/actions/delete_control_action.test.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  lazyLoadReduxEmbeddablePackage,
+  ReduxEmbeddablePackage,
+} from '@kbn/presentation-util-plugin/public';
+import { ErrorEmbeddable } from '@kbn/embeddable-plugin/public';
+
+import { ControlOutput } from '../../types';
+import { ControlGroupInput } from '../types';
+import { pluginServices } from '../../services';
+import { DeleteControlAction } from './delete_control_action';
+import { OptionsListEmbeddableInput } from '../../options_list';
+import { controlGroupInputBuilder } from '../control_group_input_builder';
+import { ControlGroupContainer } from '../embeddable/control_group_container';
+import { OptionsListEmbeddable } from '../../options_list/embeddable/options_list_embeddable';
+
+let container: ControlGroupContainer;
+let embeddable: OptionsListEmbeddable;
+let reduxEmbeddablePackage: ReduxEmbeddablePackage;
+
+beforeAll(async () => {
+  reduxEmbeddablePackage = await lazyLoadReduxEmbeddablePackage();
```

Comment:
Would it be quicker to use a mocked version of this? I am not entirely sure if one exists yet, but if not I will be adding it in https://github.com/elastic/kibana/pull/150121

Reply:
It probably would be, yeah! We don't seem to have the mock yet, though - if you plan to add one, that would be awesome!